server/controllers/index.js

```diff
@@ -2,8 +2,11 @@ const express = require('express');
 
 const router = express.Router();
 const customerInfo = require('./postCustomer');
-const availableTime = require('./availableTime');
+const timeSlots = require('./getTimeSlots');
+const getReservationData = require('./getReservationData');
 
 router.post('/api/questions/user-info', customerInfo);
-router.post('/api/availabletime', availableTime);
+router.get('/api/timeslots', timeSlots);
+router.post('/api/getReservationData', getReservationData);
```

Comment:
Why POST? It would be more conventional to use a GET with url parameters here eg `/api/getReservationData?startDate=abc&endDate=xyz

Reply:
I'd also call this endpoint `/api/reservations` for consistency (the get will then be implicit)